bundles/CoreBundle/Migrations/Version20211209131141.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Log\Handler\ApplicationLoggerDb;
+
+/**
+ * @internal
+ */
+class Version20211209131141 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable(ApplicationLoggerDb::TABLE_NAME);
+        if (!$table->hasIndex('maintenanceChecked')) {
+            $table->addIndex(['maintenanceChecked'], 'maintenanceChecked');
```

Comment:
Add to `install.sql` https://github.com/pimcore/pimcore/blob/8c8bd878bfa2e115532d5075f60a57bb6c4bf504/bundles/InstallBundle/Resources/install.sql#L5

Reply:
Done.